st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -349,6 +349,7 @@ def run(self, cmd, timeout=None, quote=False):
         chan = transport.open_session()
 
         start_time = time.time()
+        chan.get_pty()
```

Comment:
>Does the "interleaved streams" issue still exist with pty on?

There will be no stderr with pty on as you can see in fabric logs. But fabric turns it on only for sudo actions.